server/testutils/factories/referralSummary.ts

```diff
@@ -21,8 +26,32 @@ export default Factory.define<ReferralSummary, ReferralSummaryTransientParams>((
       audience => audience.value,
     ),
     courseName: `${StringUtils.convertToTitleCase(faker.color.human())} Course`,
+    earliestReleaseDate: indeterminateSentence ? tariffExpiryDate : paroleEligibilityDate || conditionalReleaseDate,
+    organisationId: faker.string.alpha({ casing: 'upper', length: 3 }),
+    prisonName: Object.prototype.hasOwnProperty.call(params, 'prisonName')
```

Comment:
This is so that we can explicitly pass in `undefined` as a value

Reply:
ah, in that case, might be worth adding this to the commit message and/or encapsulating this in a private method like getPrisonNameOrUndefinedFromParams (or something less horrible) to make that obvious

Reply:
This is actually asking "does this property exist on the object?" (differentiating `undefined` and not defined), rather than "get me the value even if undefined" - the latter (in this ternary context) would result in the same as what you wrote above:

```ts
params.prisonName || `${faker.location.county() (HMP)
```

We could add a factory helper method along the lines of `objectHasProperty(object, property)` although it might just be rewording the built-in library code?